src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.html

```diff
@@ -109,27 +109,28 @@
   </div>
 </div>
 
-<sui-modal [mustScroll]="true" [transitionDuration]="0" [size]="'normal'" class="sb-modal sb-filter-modal"
-  (dismissed)="filterModalPopup(false)" *ngIf="filterPopup">
-  <div class="sb-filter">
-    <div class="filter-select-title mb-0 px-20 mt-20 py-0"> {{ resourceService?.frmelmnts?.lbl?.selectFilters }} </div>
-    <div class="sb-filter__container">
-      <div class="sb-filter__content">
-        <app-filter [filterType]="filterType" [resetFilters]="resetFilters" [hideElements]="hideElements"
-          [selectedFilter]="selectedFilters" [filters]="filters" (filterChanged)="filterChanged($event)"
-          [chartData]="getChartData()" [telemetryInteractObject]="telemetryInteractObject">
-        </app-filter>
-      </div>
+<ng-template #filterPopUpMat >
+  <mat-dialog-actions align="left" >
+    <div class="sb-filter mb-10px">
```

Comment:
Main content should be inside mat-dialog-content directive instead of actions.

Reply:
resolved